libs/expo/betterangels/src/lib/screens/AddEditClient/index.tsx

```diff
@@ -37,6 +38,37 @@ import PersonalInfo from './PersonalInfo';
 import RelevantContacts from './RelevantContacts';
 import VeteranStatus from './VeteranStatus';
 
+const defaultSocialMedias = [
```

Comment:
on the ticket and mocks I see only 4 of these being used. Not sure what's actually intended though

Reply:
fixed

Reply:
It's still 5. Don't think it uses WhatsUp

Reply:
removed